examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -507,6 +507,14 @@ void SetupPretendDevices()
     AddEndpoint("Door 2");
     AddCluster("Door");
     AddAttribute("State", "Closed");
+
+    AddDevice("Occupancy Sensor");
+    AddEndpoint("External");
+    AddCluster("Occupancy Sensor");
+    AddAttribute("Occupancy", "1");
+    uint8_t attributeValue = 1;
+    emberAfWriteServerAttribute(1, ZCL_OCCUPANCY_SENSING_CLUSTER_ID, ZCL_OCCUPANCY_ATTRIBUTE_ID, (uint8_t *) &attributeValue,
+                                ZCL_BITMAP8_ATTRIBUTE_TYPE);
```

Comment:
Please use Accessors.h functions for writing attributes.

Reply:
Why is the value written? For demonstration?

Reply:
Instead should we read the default value and then call `AddAttribute("Occupancy", value);`?

Reply:
@dhrishi Yes for demo. We could do that.

Reply:
@dhrishi getting the default value didn't work. so i just set i to occupied for now

Reply:
Right. The default value would  be possible/applicable for clusters like on-off, level control